pyartcd/pyartcd/pipelines/scan_fips.py

```diff
@@ -40,6 +41,48 @@ def __init__(self, runtime: Runtime, data_path: str, all_images: bool, nvrs: Opt
         self.slack_client = self.runtime.new_slack_client()
         self.slack_client.bind_channel("#art-release")
 
+    async def get_pipelinerun_name(self) -> Optional[str]:
+        """
+        Returns the name of the currently running fips-pipeline PipelineRun
+        """
+        if self.taskrun_namespace is None or self.taskrun_name is None:
+            return None
+
+        cmd = [
```

Comment:
Why do we need the TaskRun? Can't we craft the PipelineRun URL and share that back? 

i.e. share https://console-openshift-console.apps.artc2023.pc3z.p1.openshiftapps.com/k8s/ns/art-cd/tekton.dev~v1~PipelineRun/fips-pipeline-run-g6lz4/ in the Slack message for example